javascript/javascript/JavaScriptParser.g4

```diff
@@ -186,15 +186,15 @@ varModifier  // let, const - ECMAScript 6
     ;
 
 continueStatement
-    : Continue ({this.notLineTerminator()}? identifier)? eos
+    : Continue ({!this.lineTerminatorAhead()}? identifier)? eos
```

Comment:
Antlr4 doesn't have a target independent action (yet). So, `!` cannot work because the syntax for negating a boolean value is different across targets. You'll have to move the "!" into the target-specific code, and name the function appropriately.

Reply:
Thank you! Fixed it